<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>STFTWgtClass</class>
 <widget class="QWidget" name="STFTWgtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1293</width>
    <height>991</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>STFTWgt</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_STFTPara">
      <attribute name="title">
       <string>分析参数</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>原始信号</string>
           </property>
           <layout class="QVBoxLayout" name="VLayout_Cht_Sig"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>原始信号导入</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QGroupBox" name="groupBox_SigFrom">
              <property name="title">
               <string>来源（当前信号）</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QRadioButton" name="radioBtn_SigFrom_Gen">
                 <property name="text">
                  <string>合成信号</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioBtn_SigFrom_Wav">
                 <property name="text">
                  <string>文件导入</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_GenSig">
                <property name="text">
                 <string>合成信号源</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_GenSig">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_WavSig">
                <property name="text">
                 <string>文件信号源</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_WavSig">
                <property name="minimumSize">
                 <size>
                  <width>120</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>120</width>
                  <height>16777215</height>
                 </size>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_SigChnNum">
              <property name="text">
               <string>通道数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_SigDuration">
              <property name="text">
               <string>持续时间：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_SigPotintCount">
              <property name="text">
               <string>载入点数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_SigSampleRate">
              <property name="text">
               <string>采样率：</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QLabel" name="label_dataLoadState">
                <property name="text">
                 <string>导入状态：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="progressBar_dataLoadState">
                <property name="value">
                 <number>24</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="btn_loadSig">
              <property name="text">
               <string>导入</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>320</height>
            </size>
           </property>
           <property name="title">
            <string>窗形</string>
           </property>
           <layout class="QVBoxLayout" name="VLayout_Cht_STFTWnd">
            <item>
             <widget class="QWidget" name="widget" native="true"/>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>时域窗型控制</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="groupBox_8">
              <property name="title">
               <string>形状</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QRadioButton" name="radioBtn_Wnd_Null">
                 <property name="text">
                  <string>无</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioBtn_Wnd_Hann">
                 <property name="text">
                  <string>Hann</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_9">
              <property name="title">
               <string>长度定义</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QRadioButton" name="radioBtn_WndLengthType_Point">
                 <property name="text">
                  <string>点数</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioBtn_WndLengthType_Time">
                 <property name="text">
                  <string>时长</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_WndLengthPoint">
                <property name="text">
                 <string>窗型点数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SBox_WndLengthPointCount">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>4096</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLabel" name="label_WndLength_Time">
                <property name="text">
                 <string>窗型时长(ms)：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="DSBox_WndLength_TimeMSec">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <double>100000.000000000000000</double>
                </property>
                <property name="value">
                 <double>100.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_WndSampleRate">
                <property name="text">
                 <string>采样率</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SBox_WndSampleRate">
                <property name="maximum">
                 <number>100000</number>
                </property>
                <property name="value">
                 <number>44100</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_resultWndLength_Point">
              <property name="text">
               <string>窗型等价点数：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_resultWndLength_Time">
              <property name="text">
               <string>窗型等价时长：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_GenWnd">
              <property name="text">
               <string>生成窗型</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>STFT结果</string>
           </property>
           <layout class="QVBoxLayout" name="VLayout_STFTResult_Hot"/>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="minimumSize">
            <size>
             <width>240</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>240</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="title">
            <string>STFT控制</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_17">
              <item>
               <widget class="QLabel" name="label_FFT_source">
                <property name="text">
                 <string>FFT来源</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="combox_FFTSource"/>
              </item>
              <item>
               <widget class="QCheckBox" name="checkBox_STFT_MultiThreadMode">
                <property name="text">
                 <string>多线程模式</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_STFTGapType">
              <property name="title">
               <string>步长类型</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_13">
               <item>
                <widget class="QRadioButton" name="radioBtn_STFTGapType_WndRatio">
                 <property name="text">
                  <string>窗口比例</string>
                 </property>
                 <property name="checked">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QRadioButton" name="radioBtn_STFTGapType_Time">
                 <property name="text">
                  <string>时间长度</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="label_WndLength_Time_2">
                <property name="text">
                 <string>步长时间(ms)：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="DSBox_STFTGapLength_timeMSec">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="maximum">
                 <double>100000.000000000000000</double>
                </property>
                <property name="value">
                 <double>20.000000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_15">
              <item>
               <widget class="QLabel" name="label_WndLengthPoint_2">
                <property name="text">
                 <string>窗口比例</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QDoubleSpinBox" name="DSBox_STFTGapWndLenghtRatio">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="maximum">
                 <double>1.000000000000000</double>
                </property>
                <property name="singleStep">
                 <double>0.010000000000000</double>
                </property>
                <property name="value">
                 <double>0.250000000000000</double>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_STFTGapInTime">
              <property name="text">
               <string>步长等价时间：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_STFTGapInWndRatio">
              <property name="text">
               <string>步长等价窗口比例：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QCheckBox" name="checkBox_genWndSpecConstrainAfterSTFTResult">
              <property name="text">
               <string>之后生成窗谱抑制输出</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <widget class="QLabel" name="label_dataLoadState_2">
                <property name="text">
                 <string>生成状态：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QProgressBar" name="progressBar_STFTGenerateState">
                <property name="value">
                 <number>24</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="label_processState">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>20</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>20</height>
               </size>
              </property>
              <property name="text">
               <string>活跃线程：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="btn_genSTFTResult">
              <property name="text">
               <string>生成STFT结果</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_detailShow">
      <attribute name="title">
       <string>详细数据</string>
      </attribute>
      <layout class="QVBoxLayout" name="VLayout_STFT_detail"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
